creativecloud/scripts/utils.js

```diff
@@ -136,6 +134,7 @@ export async function acomsisCookieHandler() {
     }
     return isSignedInUser;
   }
+  
```

Comment:
@robert-bogos Could you please remove trailing space from this line? It is causing eslint error.

Reply:
Done 👍